views/hello-world.jsx

```diff
@@ -111,155 +74,38 @@ const FormDefaultExample = () => {
-    const quotaData = await clientApi.queryTokenUsage()
-    if (quotaData) {
-      const { max_quota, quota_used } = quotaData
-      setTokenUsageRatio((quota_used / max_quota).toFixed(2))
     }
-  }
-
-  useEffect(() => {
-    (async () => {
-      await getTokenUsageRatio()
-    })()
-  }, [])
+  }, [sessions]);
 
   return (
     <Page>
       <DebugComponent/>
       <Wrapper>
-        <LeftBox>
-          <PreDefinedPrompts onSelect={handleSelect}/>
-          {
-            1 - tokenUsageRatio <= WARNING_USAGE && <QuotaBox>
-              <SectionMessage
-                appearance="warning"
-                // TODO: add topup action button
-                actions={[
-                  <SectionMessageAction key='topup' onClick={() => window.open('https://zenuml.atlassian.net/servicedesk/customer/portals', '_blank', 'noreferrer')}>Top Up</SectionMessageAction>
-                ]}>
-                <div>
-                  <p style={{marginBottom: '6px'}}>Token usage below {(1 - tokenUsageRatio) * 100}%</p>
-                  <ProgressBar ariaLabel={`Remains {(1 - tokenUsageRatio) * 100}%`} value={1 - tokenUsageRatio} />
-                </div>
-              </SectionMessage>
-            </QuotaBox>
-          }
-        </LeftBox>
-        <ChatBox>
-          <Form onSubmit={onSubmit}>
-            {({formProps, submitting}) => (
-              <form {...formProps}>
-                <FormHeader title='Your Prompts'/>
-                <FormSection>
-                  <Field name='query'>
-                    {({fieldProps}) => (
-                      <Fragment>
-                        <TextArea
-                          ref={inputRef}
-                          resize='smart'
-                          value={currentPrompt}
-                          placeholder='e.g. Write a Job Description for Senior DevOps Engineer'
-                          {...fieldProps}
-                        />
-                        <HelperMessage>
-                          Go to{' '}
-                          <a target='_blank' rel="noreferrer"
-                            href={`https://github.com/ZenGPT/confluence-gpt/wiki/Crafting-Effective-Prompts`}
-                          >
-                            Crafting Effective Prompts
-                          </a>{' '}
-                          for more information.
-                        </HelperMessage>
-                      </Fragment>
-                    )}
-                  </Field>
-                </FormSection>
-
-                <FormFooter>
-                  <ButtonGroup>
-                    <LoadingButton
-                      type='submit'
-                      appearance='primary'
-                      isLoading={submitting}
-                    >
-                      Submit
-                    </LoadingButton>
-                    <Button type={'button'} appearance={'subtle'} onClick={() => {AP.dialog.close()}}>Close</Button>
```

Comment:
Close button is needed somewhere on the page.

Reply:
Added:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/125171539/232965642-613b51cc-75c2-4d9b-867b-6af83621472d.png">